internal/storage/leveldb.go

```diff
@@ -236,6 +295,47 @@ func saveIntoErrorQueue(t *leveldb.Transaction, neutronTXHash string, txInfo rel
 	return nil
 }
 
+func removeFromUnsuccessfulQueue(t *leveldb.Transaction, queryID uint64, tXHash string) error {
+	key := constructUnsuccessfulQueueKey(queryID, tXHash)
+	err := t.Delete(key, nil)
+	if err != nil {
+		return fmt.Errorf("failed to remove UnsuccessfulTxInfo under the key %d%s: %w", queryID, tXHash, err)
```

Comment:
`under the key %d%s: %w", queryID, tXHash` is really going to look mad in the service logs. please make the description more clear. no need to let the user know the way the storage constructs keys